Quaver.Shared/Screens/Result/ResultScreen.cs

```diff
@@ -161,6 +161,9 @@ public ResultScreen(Replay replay)
         /// </summary>
         public override void OnFirstUpdate()
         {
+            if (!AudioEngine.Track.IsDisposed && AudioEngine.Track.IsPlaying)
+                AudioEngine.Track.Pause();
```

Comment:
I don't think this is the correct thing to do. If the song is still playing after the map is done, let it play until the end. Besides, this also abruptly pauses the song when opening a score in song select.